app/src/main/java/com/amaze/filemanager/asynchronous/services/CopyService.java

```diff
@@ -257,7 +257,12 @@ protected Void doInBackground(Bundle... p1) {
 
             watcherUtil = new ServiceWatcherUtil(progressHandler);
 
-            addFirstDatapoint(sourceFiles.get(0).getName(), sourceFiles.size(), totalSize, move);
+            try {
+                addFirstDatapoint(sourceFiles.get(0).getName(), sourceFiles.size(), totalSize, move);
+            } catch (IllegalStateException e) {
```

Comment:
This `Exception` shouldn't be catched, as it should never be thrown. The underlying bug should be fixed.

Reply:
The exception shouldn't be raised after this fix. I put this code for ease of testing. Can remove it if you want.